docs/usage/index.md

```diff
@@ -0,0 +1,119 @@
+# Using Pyodide
+
+Pyodide may be used in any context where you want to run Python inside a web
+browser or a backend JavaScript environment.
+
+## Web browsers
+
+To use Pyodide on a web page you need to load `pyodide.js` and initialize
+Pyodide with {any}`loadPyodide` specifying a index URL for packages:
+
+```html-pyodide
+<!DOCTYPE html>
+<html>
+  <head>
+      <script src="https://cdn.jsdelivr.net/pyodide/v0.18.0/full/pyodide.js"></script>
+  </head>
+  <body>
+    <script type="text/javascript">
+      async function main(){
+        let pyodide = await loadPyodide({
+          indexURL : "https://cdn.jsdelivr.net/pyodide/v0.18.0/full/"
+        });
+        console.log(pyodide.runPython("1 + 2"));
+      }
+      main();
+    </script>
+  </body>
+</html>
+```
+
+See the {ref}`quickstart` for a walk through tutorial as well as
+{ref}`loading_packages` and {ref}`type-translations` for more in depth
+discussion about existing capabilities.
+
+You can also use the [Pyodide NPM
+package](https://www.npmjs.com/package/pyodide) to integrate Pyodide into your
+application.
+
+```{note}
+To avoid confusion, note that:
+ - `https://cdn.jsdelivr.net/pyodide/` distributes Python packages built with Pyodide as well as `pyodide.js`
+ - `https://cdn.jsdelivr.net/npm/pyodide@0.18.0/` is a mirror of the Pyodide NPM package, which includes none of the WASM files
+```
+
+### Supported browsers
+
+Pyodide works in any modern web browser with WebAssembly support.
+
+**Tier 1** browsers are tested as part of the test suite with continuous integration,
```

Comment:
Actually I'll remove this section from this PR as it requires more bisection to find the minimal version. For instance Firefox 60 doesn't work due to not supporting globalThis. Will open a separate PR.